lando/server/jobapi.py

```diff
@@ -181,14 +180,13 @@ def get_input_files(self):
 
     def get_credentials(self):
         """
-        Get all dds user/app credentials attached to this job.
-        :return: Credentials: let's user lookup credential info based on bespin user/app ids.
+        Get all bespin service account credentials.
+        :return: Credentials: bespin DukeDS credentials
         """
         job = self.get_job()
-        user_id = job.user_id
         credentials = Credentials()
 
-        for user_credential_data in self.api.get_dds_user_credentials(user_id):
+        for user_credential_data in self.api.get_dds_user_credentials():
```

Comment:
It might be more clear if we renamed `DDSUserCredential` to `DDSBespinWorkerUserCredential` but that's rather long.